storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/JdbcQueryTemplater.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.storage.connectors.jdbc.retriever;
+
+import feast.proto.core.FeatureSetProto;
+import feast.proto.serving.ServingAPIProto;
+import feast.storage.api.retriever.FeatureSetRequest;
+import java.sql.Timestamp;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+public interface JdbcQueryTemplater {
```

Comment:
Shall we move this interface into the `common` folder, and move `SnowflakeQueryTemplater` class into the `snowflake` folder?

Reply:
`common` is a word that should be banned in software naming.  `feast.storage.connectors.jdbc` is a fine place for this.